contracts/InterchainTokenFactory.sol

```diff
@@ -446,6 +447,32 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
         tokenId = deployRemoteCanonicalInterchainToken(originalTokenAddress, destinationChain, gasValue);
     }
 
+    // TODO: Should we reuse interchainTokenDeploySalt?
```

Comment:
```suggestion
```
removed since it can be problematic for trustlessly deployed tokens if the deployer key is compromised